var/spack/repos/builtin/packages/http-get/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class HttpGet(MakefilePackage):
+    """Http_get fetches an HTTP URL and dumps the contents to stdout.
+    It does not do gopher, ftp, file, news, or any other type of URL,
+    only HTTP. It can be configured to do HTTPS fetches as well."""
+
+    homepage = "http://www.acme.com/software/http_get/"
+    url      = "http://www.acme.com/software/http_get/http_get_23May2018.tar.gz"
+
+    version('23May2018', sha256='7d46ce25e53b6d3e27a99c1853c3054a046cc97d5e30a713a7ec986cfe7c4fe0')
```

Comment:
This will be problematic if anyone ever adds another version, as the version won't sort correctly. I would change the version of `2018-05-23` and either manually specify the URL for each version or use Python's datetime module to convert from one format to another in `url_for_version`. This is a suggestion, not a requirement, it could wait until someone actually needs another version.